src/dev.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import styles from './dev.css'
```

Comment:
why append to the body manually? any reason to not just do another `ReactDOM.render()` here?